src/app/exercise4/game-control/game-control.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-game-control',
+  templateUrl: './game-control.component.html',
+  styleUrls: ['./game-control.component.scss']
+})
+export class GameControlComponent implements OnInit {
+  @Output() handleInterval = new EventEmitter<number>()
+  timer: ReturnType<typeof setTimeout> = setTimeout(() => '', 1000);
+  count:number = 0
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  onStart(){
+    this.timer = setInterval(()=>{
+        this.handleInterval.emit(this.count+1)
+        this.count++
```

Comment:
Instead of  this.count+1on line 20 and then on line 21 this.count++, you can use this.count+=1 or ++this.count. It will increment and assign the new value.

Here is a reference for you:
https://codeburst.io/javascript-increment-and-decrement-8c223858d5ed